### PR TITLE
Update soak frequency

### PR DESCRIPTION
**Pull request by jasonmci**
## Solution

### CODE REVIEW

File: integration-tests/ccip-tests/testconfig/override/mainnet.toml [hunk 1]
```diff
@@ -701,34 +701,34 @@ TestRunName = 'mainnet-2.7-ccip1.2'
...
-  'BASE_MAINNET,POLYGON_MAINNET', # added as batch 2
-  'BSC_MAINNET,OPTIMISM_MAINNET', # added as batch 2
-  'AVALANCHE_MAINNET,BASE_MAINNET', # added as batch 2
-  'BSC_MAINNET,WEMIX_MAINNET', # added as batch 2
-  'AVALANCHE_MAINNET,WEMIX_MAINNET', # added as batch 2
-  'POLYGON_MAINNET,WEMIX_MAINNET', # added as batch 2
-  'WEMIX_MAINNET,ARBITRUM_MAINNET', # added as batch 2
-  'OPTIMISM_MAINNET,WEMIX_MAINNET' # added as batch 2
+  'ETHEREUM_MAINNET,BASE_MAINNET'
+#  'ETHEREUM_MAINNET,WEMIX_MAINNET',
```
Comment (AnieeG) [inline integration-tests/ccip-tests/testconfig/override/mainnet.toml]: is this commenting intentional?

File: integration-tests/ccip-tests/testconfig/override/mainnet.toml [hunk 2]
```diff
@@ -678,11 +677,12 @@ NetworkPairs = [
...
-  'AVALANCHE_MAINNET,WEMIX_MAINNET', # added as batch 2
-  'POLYGON_MAINNET,WEMIX_MAINNET', # added as batch 2
-  'WEMIX_MAINNET,ARBITRUM_MAINNET', # added as batch 2
-  'OPTIMISM_MAINNET,WEMIX_MAINNET' # added as batch 2
+ 'WEMIX_MAINNET,KROMA_MAINNET',
+ 'BSC_MAINNET,WEMIX_MAINNET', # added as batch 2
+ 'AVALANCHE_MAINNET,WEMIX_MAINNET', # added as batch 2
+ 'POLYGON_MAINNET,WEMIX_MAINNET', # added as batch 2
+ 'WEMIX_MAINNET,ARBITRUM_MAINNET', # added as batch 2
+ 'OPTIMISM_MAINNET,WEMIX_MAINNET' # added as batch 2
```
Comment (AnieeG) [inline integration-tests/ccip-tests/testconfig/override/mainnet.toml]: indentation

**Review by AnieeG — COMMENTED** [review]
Are all these network pair changes intentional?

File: integration-tests/ccip-tests/testconfig/override/mainnet.toml (hunk 1 above)
Comment (jasonmci) [reply integration-tests/ccip-tests/testconfig/override/mainnet.toml]: fixed. I will make copies for short term changes from now on.

File: integration-tests/ccip-tests/testconfig/override/mainnet.toml (hunk 2 above)
Comment (jasonmci) [reply integration-tests/ccip-tests/testconfig/override/mainnet.toml]: fixed

**Review by AnieeG — APPROVED** [review]

**Pull request merged**